robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -217,20 +246,29 @@ def validate_field_values_under_section(self, section=None, **kwargs):
                     label, "contains", value, section
                 )
 
-    def get_next_payment_date_number(self, paynum):
+    def get_next_payment_date_number(self, paynum, format=None):
```

Comment:
How about defaulting "format=True" ? I think that would be more understandable since format=True represents the date gets formatted before being returned

Reply:
Agree and will update it